validate.py

```diff
@@ -18,7 +18,6 @@
 
 # Global variables
 
-logging.basicConfig(level="DEBUG")
```

Comment:
When I changed the `LOG_LEVEL` to "INFO", I wasn't seeing the `logger.info` calls from `execute_query_and_write_to_csv`. Changing them to `warning` made them show up. It seems to work as expected if you call this `basicConfig` method. So maybe we can keep this line but just use `logging.basicConfig()`? You probably know better how the `logging` module works than I do, but it was odd to me that I couldn't get the info messages to show up.

Reply:
Ok adding in logging.baseConfig looks like it gets this working as expected. 
WARN works the same as WARNING. It looks like they're both supported but WARN was deprecated so updating that in the sample too.